### PR TITLE
Add documentation

### LINKED ISSUE #36: Add documentation
- installation
- quick start
- explain serialization vs encoder
- datatype support in serializer
- contributing
- mention semver
- license

### PR DESCRIPTION
**Pull request by dincho**
closes #36

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+4. If you've changed APIs, update the documentation.
+5. Ensure the test suite passes.
+6. Make sure your code lints.
+7. Issue that pull request!
+
+## Any contributions you make will be under the ISC Software License
+In short, when you submit code changes, your submissions are understood to be under the same [ISC License](https://choosealicense.com/licenses/isc/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/aeternity/aepp-calldata-js/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
```
Comment (marc0olo) [inline CONTRIBUTING.md:28]: I guess the link for opening a new issue is missing here, right?

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "aepp-calldata-js",
+  "name": "@aeternity/aepp-calldata",
   "version": "0.0.1",
   "description": "Aeternity data serialization library",
-  "main": "Encoder.js",
+  "main": "src/Encoder.js",
   "keywords": "aeternity, aesophia, fate, calldata, serialization, encoder, decoder",
```
Comment (marc0olo) [inline package.json:6]: this isn't a change. but my IDE tells me that the list of keywords should be an array

**Review by marc0olo — CHANGES_REQUESTED** [review]
the link to open a new issue should be fixed

File: package.json (hunk 2 above)
Comment (dincho) [reply package.json]: Fair point, somehow related to the docs anyway, fixed in 7d64be5

File: CONTRIBUTING.md (hunk 1 above)
Comment (dincho) [reply CONTRIBUTING.md]: in c550ba7

**Review by marc0olo — APPROVED** [review]
looks good to me now

**Pull request merged**